bot/cogs/moderation.py

```diff
@@ -181,12 +198,142 @@ async def clear(self, ctx: Context, amount: int) -> None:
                 **Amount**: {amount}
                 """
             ),
-            color=discord.Color.orange(),
+            color=Color.orange(),
         )
         message = await ctx.send(ctx.author.mention, embed=embed)
         await asyncio.sleep(2.5)
         await message.delete()
 
+    @command()
+    @has_permissions(manage_messages=True)
+    async def shift(self, ctx, count: int, target: TextChannel, copy: bool = False) -> None:
+        """Copy or Move specified messages amount to specified channel"""
+        if not (5 <= count <= 150):
+            await ctx.send("Amount of messages shifted must be greater than 0 and smaller than 150")
+            return
```

Comment:
You can make that only one line (that's the behaviour recommended on the `discord.py` server)
That goes for every repetition of this pattern